pkg/cloudprovider/aws/block_storage_adapter.go

```diff
@@ -68,18 +75,19 @@ func (op *blockStorageAdapter) GetVolumeInfo(volumeID string) (string, *int, err
 
 	var (
 		volumeType string
-		iops       int
+		iops       *int
 	)
 
 	if vol.VolumeType != nil {
 		volumeType = *vol.VolumeType
 	}
 
-	if vol.Iops != nil {
-		iops = int(*vol.Iops)
+	if iopsVolumeTypes.Has(volumeType) && vol.Iops != nil {
+		iopsVal := int(*vol.Iops)
```

Comment:
If we make the change above in backup.go to `*int64`, this logic can be simplified:

```go
var iops *int64
if iopsVolumeTypes.Has(volumeType) && vol.Iops != nil {
  iops = vol.Iops
}
```

right?